### DISCUSSION TITLE
Optimizing object detection notification time - general recommendations

### DISCUSSION THREAD

**Discussion by cchewanik** [blakeblackshear/frigate#6908] [General]
Hi, looking for some general recommendations on how to optimize my config to get an MQTT object recognized message as quick as possible. For example I've noticed that the go2rtc stream lags compared to using a direct connection with rtsp.

Are there any other config options that have an impact on processing time? Ex. Enabling Birdseye, having lots of zones per camera, etc.

Thanks!

**Comment by NickM-27** [comment]
the biggest difference maker in latency is going to be ffmpeg input / output args and using hwaccel, as well as having some detector like a coral or GPU

**Reply by cchewanik** [reply]
I'm using a coral and have VAAPI enabled also. I'm running it on a Synology so I have limited resources, trying to see what else I can optimize!

Thanks for the reply and all your hard work on this project!
